bindings/julia/Displaz.jl

```diff
@@ -79,14 +117,24 @@ interpret_color(s::String) = length(s) == 1 ? interpret_color(s[1]) : error("Unk
 interpret_color(c::Char) = color_names[c]
 
 
+const shape_names = @compat Dict('.' => [0],
+                                 's' => [1],
+                                 'o' => [2],
+                                 'x' => [3],
+                                 '+' => [4],
+								 '-' =>	[5])
+
+interpret_shape(markershape) = markershape
+interpret_shape(c::Char) = shape_names[c]
+
 # True if plots are to be plotted over the previous data, false to clear before
 # plotting new data sets.
 _hold = false
 
-
 # Basic 3D plotting function for points
 function plot(position; color=[1 1 1], markersize=[0.1], markershape=[0])
```

Comment:
Can we rename this function as it conflicts with other plotting functions? Any preferences @c42f ?